src/primitives.rs

```diff
@@ -1,6 +1,67 @@
-//! This module contains type aliases for C's fixed-width integer types .
+//! This module contains type aliases for C's platform-specific types
+//! and fixed-width integer types.
 //!
-//! These aliases are deprecated: use the Rust types instead.
+//! The platform-specific types definitions were taken from rust-lang/rust in
+//! library/core/src/ffi/mod.rs and should be kept in sync while the MSRV is below 1.64.
```

Comment:
```rust
//! The platform-specific types definitions were taken from rust-lang/rust in
//! library/core/src/ffi/primitives.rs
```

The file moved, and probably no need to mention MSRV here because of the other issue.

Reply:
Done!